examples/experimental/dagster-airlift/dagster_airlift/in_airflow/mark_as_migrating.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from typing import Any, Dict
+
+from airflow import DAG
+
+from ..migration_state import AirflowMigrationState
+
+
+def mark_as_dagster_migrating(
+    *,
+    global_vars: Dict[str, Any],
+    migration_state: AirflowMigrationState,
+    logger: logging.Logger = logging.getLogger("dagster_airlift"),
```

Comment:
this should be `None` and then default to `logging.getLogger("dagster_airlift")` in the body given the way that Python does mutable parameters